src/core/components/hv-route/index.tsx

```diff
@@ -414,7 +343,13 @@ function HvRouteFC(props: Types.Props) {
   const backContext = useContext(BackBehaviorContext);
   const docContext = useContext(Contexts.DocContext);
 
-  const url = getRouteUrl(props, navigationContext);
+  const url =
+    props.navigation === undefined
+      ? navigationContext.entrypointUrl
+      : NavigatorService.cleanHrefFragment(
+          props.route?.params?.url || navigationContext.entrypointUrl,
+        );
```

Comment:
sanity check: the logic has changed to always call `NavigatorService.cleanHrefFragment` instead of conditionally returning `undefined` when `props.route?.params?.url` is falsy - is this intentional or could it have any unintended side effect?